src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -821,6 +822,19 @@ def aks_show(cmd, client, resource_group_name, name, aks_custom_headers=None):
     return _remove_nulls([mc])[0]
 
 
+def aks_stop(cmd, client, resource_group_name, name, aks_custom_headers=None, no_wait=False, yes=False):
+    headers = get_aks_custom_headers(aks_custom_headers)
```

Comment:
Please add some tests for this change.
- For UT, see examples in `test_managed_cluster_decorator.py`, you could easily mock any function call via proper patch.
- For live test, see examples in `test_aks_comands.py`.